db/wrapper.go

```diff
@@ -543,6 +511,63 @@ func (q *queryWrapper) Iter() Iterator {
 	}
 }
 
+type aggregationWrapper struct {
+	ctx      context.Context
+	coll     *mongo.Collection
+	pipeline interface{}
+	cursor   *mongo.Cursor
+	hint     interface{}
+	maxTime  time.Duration
+}
+
+func (a *aggregationWrapper) Hint(hint interface{}) Aggregation   { a.hint = hint; return a }
+func (a *aggregationWrapper) MaxTime(d time.Duration) Aggregation { a.maxTime = d; return a }
+
+func (a *aggregationWrapper) exec() error {
+	if a.cursor != nil {
+		return nil
+	}
+
+	opts := options.Aggregate().SetAllowDiskUse(true)
+	if a.hint != "" {
```

Comment:
This has to check if a.hint is non-nil.

Reply:
would it be a problem if it is nil? I would think the options would just be unchanged since hint was nil and will just be set to nil.
On the other hand, maybe I should just remove the check entirely?

Reply:
I don't think the check is necessary.

Reply:
I removed it.
For consistency I also removed it from the `queryWrapper`